crates/lair_keystore/src/lib.rs

```diff
@@ -28,6 +28,8 @@ pub async fn execute_lair() -> LairResult<()> {
     }
 
     let config = config.build();
+    
+    println!("#lair-keystore-dir:{:?}#", config.get_root_path());
```

Comment:
I wrote this because without the developer lacks general awareness of how the system is operating on the computer, such as "from where / which directory"?